tests/integration/guzzle/3/ExampleHttpClient.php

```diff
@@ -24,6 +24,25 @@ public function get($url)
 
         return null;
     }
+    
+    public function post($url, $body)
+    {
+        $client = new Client();
+        
+        try {
+            $request = $client->post($url);
+            $request->setBody($body, 'application/json');
```

Comment:
Just for me to know, is the 'application/json' parameter changing anything how Guzzle sends the request? (Except from a `Content-Type` header?